src/manage/subscriptions/index.md

```diff
@@ -7,7 +7,9 @@ Subscription details and pricing are summarised on our <MainDomainNameLink id="p
 The current *Premium* subscription system is explained in more detail in our blog <MainDomainNameLink id="blog/unlocking-premium-features-of-mergin-maps-for-all" desc="Unlocking premium features of Mergin Maps for all"/>.
 :::
 
-Subscriptions are based on the number of contributors on the workspace. *Contributors* are workspace members or guests who have Writer, Editor, Admin or Owner [access rights](../permissions/). The number of read-only users is unlimited: they do not count towards your workspace contributors.
+Subscriptions are based on the number of *contributors* on the workspace. Per each contributor seat, you get 1 GB of storage. Storage is shared across the whole workspace. 
+
+*Contributors* are workspace members or guests who have Writer, Editor, Admin or Owner [access rights](../permissions/).  The number of read-only users is unlimited: they do not count towards your workspace contributors. 
```

Comment:
Seems like there is an extra space between sentences.

`...[access rights](../permissions/).  The number of ...`